gamedata/movedefs.lua

```diff
@@ -80,8 +80,8 @@ local moveDatas = {
 	ATANK3 = {
 		crushstrength = 30,
 		depthmod = 0,
-		footprintx = 3,
-		footprintz = 3,
+		footprintx = 4,
+		footprintz = 4,
```

Comment:
Shouldn't this be called ATANK4 now? It seemed the name was intended to match the foorptinr size,